src/test/java/ca/team2706/frc/robot/commands/drivebase/MotionMagicTest.java

```diff
@@ -57,6 +58,7 @@
 
     @Before
     public void setUp() throws NoSuchFieldException, IllegalAccessException {
+        Util.resetSubsystems(); // Always do this because it's useful.
```

Comment:
So @Zinka010 this is the most common error when tests fail on linux but pass everywhere else. Need to reset all subsystems in order to make sure that any new expectations that you set (such as for the sensor collection) works.